### PR TITLE
[16.0] [MIG] product_dimension

### PR DESCRIPTION
**Pull request by FrancoMaxime**
#1157

### CODE REVIEW

File: product_dimension/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[ The change log. The goal of this file is to help readers
```
Comment (emagdalenaC2i) [inline product_dimension/readme/HISTORY.rst:1]: I think this file can be deleted

**Review by emagdalenaC2i — APPROVED** [review]
LGTM, just a little comment about the History.rst file

File: product_dimension/readme/HISTORY.rst (hunk 1 above)
Comment (rousseldenis) [reply product_dimension/readme/HISTORY.rst:1]: @emagdalenaC2i readme fragments should never be deleted.

@FrancoMaxime I think people that wrote that file forgot to remove unnecessary comments

File: product_dimension/readme/HISTORY.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  understand changes between version. The primary audience is
+  end users and integrators. Purely technical changes such as
+  code refactoring must not be mentioned here.
+
+  This file may contain ONE level of section titles, underlined
+  with the ~ (tilde) character. Other section markers are
+  forbidden and will likely break the structure of the README.rst
+  or other documents where this fragment is included. ]
```
Comment (rousseldenis) [inline product_dimension/readme/HISTORY.rst:9]:
```suggestion

```

File: product_dimension/readme/HISTORY.rst (hunk 1 above)
Comment (rousseldenis) [reply product_dimension/readme/HISTORY.rst:1]: But in fact, after having read it, the section could be kept.

Comment (FrancoMaxime) [reply product_dimension/readme/HISTORY.rst:1]: The history file has been updated.  And a new run of oca-gen-addon-readme has been made.

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**